CMakeLists.txt

```diff
@@ -121,7 +121,7 @@ set (WAMR_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
 include (${WAMR_ROOT_DIR}/build-scripts/runtime_lib.cmake)
 
-set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Wformat -Wformat-security -Wshadow -Wno-unused-parameter")
+set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Wformat -Wformat-security -Wshadow -Wno-unused-parameter -fvisibility=hidden")
```

Comment:
How about also adding the flag in `product-mini/platforms/linux/CMakeLists.txt`?
https://github.com/bytecodealliance/wasm-micro-runtime/blob/058bc47102ea0a97159ef7f9c4c1d471af6a8e9a/product-mini/platforms/linux/CMakeLists.txt#L148